app/component/ItineraryPage.js

```diff
@@ -774,6 +774,15 @@ class ItineraryPage extends React.Component {
     });
   };
 
+  focusToStep = (lat, lon) => {
+    this.setState({
+      bounds: [
+        [lat, lon],
+        [lat, lon],
+      ],
+    });
+  };
+
```

Comment:
In which way focusToStep differs from focusToPoint?

Reply:
You are right, it's basically the same. I will remove that function.